### PR TITLE
LPD-24389 - Hide Groovy and Java options form Workflow Notifications

### PR DESCRIPTION
**Pull request by danvb112**
Related issue: https://liferay.atlassian.net/browse/LPD-24389

### CODE REVIEW

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/util/detectGroovyOrJavaScript.js [hunk 1]
```diff
@@ -47,6 +48,36 @@ function hasScriptActionsOrAssignments(data) {
 	}
 }
 
+function isNumericKey(key) {
+	return !isNaN(parseFloat(key)) && isFinite(key);
+}
+
+function hasScriptNotificationRecipient(data) {
+	if (data.notifications) {
```
Comment (cgmonte) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/util/detectGroovyOrJavaScript.js:56]: Hey, great, no recursion was needed.

**Review by cgmonte — APPROVED** [review]
LGTM!

**Comment by danvb112** [comment]
ci:test:workflow

**Comment by danvb112** [comment]
ci:test:workflow-metrics

**Comment by danvb112** [comment]
CI failures are not related!

**Comment by danvb112** [comment]
ci:forward:force

**Comment by danvb112** [comment]
ci:test:relevant

**Comment by danvb112** [comment]
Sent to: https://github.com/brianchandotcom/liferay-portal/pull/149761

**Pull request closed without merging**